index.js

```diff
@@ -1,9 +1,25 @@
 const express = require('express');
-
 const server = express();
+const db = require('./data/db.js')
+
+
+server.get('/api/users', (req, res) => {
+  db.find()
+  .then(users => {
+            res.send({ users });
+        })
+  .catch(error => {
+    res.status(500).send({ error: "The users information could not be retrieved." })
+  })
+});
+
+
+
+server.post("/api/users", (req, res) => {
 
-server.get('/', (req, res) => {
-  res.send('Hello World');
+  if (req.query.name == "" || req.query.bio == ""){
+  res.status(400).send({ errorMessage: "Please provide name and bio for the user."});
```

Comment:
Possibly a better place for this error checking and message would be in the `post` route 😄